docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1240,6 +1253,8 @@ Here's how it works:
 
 Rules with `$cookie` modifier are not supported by AdGuard Content Blocker, AdGuard for iOS, and AdGuard for Safari.
 
+In [AdGuard Browser Extension for Chrome MV3](/general/adguard-browser-extension/mv3-version) we delete cookies in 2 ways: from `content-script` side (to which we have access) and from `onBeforeSendHeaders` listener. Since `onBeforeSendHeaders` and other listeners are no longer blocking, we are not able to delete them in all cases. You can check if a rule works with [this test](https://testcases.agrd.dev/Filters/cookie-rules/test-cookie-rules).
```

Comment:
Here and in all other places, it will be more consistent with the document to use `[AdGuard for Chrome MV3][ext-mv3]` and only from that section you link the MV3 page.